public/app/features/commandPalette/actions/staticActions.ts

```diff
@@ -142,7 +142,7 @@ export function useStaticActions(): CommandPaletteAction[] {
     if (shouldRenderInviteUserButton) {
       navBarActions.push({
         id: 'invite-user',
-        name: t('navigation.invite-user.invite-new-member-button', 'Invite new member'),
+        name: t('navigation.invite-user.invite-new-user-button', 'Invite new user'),
```

Comment:
I think it's fine to leave this in command palette as is a "nice to have" feature that makes it slightly more usable / better